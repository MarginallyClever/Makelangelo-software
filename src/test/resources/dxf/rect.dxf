  0
SECTION
  2
HEADER
  9
$ACADVER
  1
AC1014
  9
$HANDSEED
  5
FFFF
  9
$MEASUREMENT
 70
     1
  0
ENDSEC
  0
SECTION
  2
TABLES
  0
TABLE
  2
VPORT
  5
8
330
0
100
AcDbSymbolTable
 70
     4
  0
VPORT
  5
2E
330
8
100
AcDbSymbolTableRecord
100
AcDbViewportTableRecord
  2
*ACTIVE
 70
     0
 10
0.0
 20
0.0
 11
1.0
 21
1.0
 12
210.0
 22
148.5
 13
0.0
 23
0.0
 14
10.0
 24
10.0
 15
10.0
 25
10.0
 16
0.0
 26
0.0
 36
1.0
 17
0.0
 27
0.0
 37
0.0
 40
341.0
 41
1.24
 42
50.0
 43
0.0
 44
0.0
 50
0.0
 51
0.0
 71
     0
 72
   100
 73
     1
 74
     3
 75
     0
 76
     0
 77
     0
 78
     0
  0
ENDTAB
  0
TABLE
  2
LTYPE
  5
5
330
0
100
AcDbSymbolTable
 70
     1
  0
LTYPE
  5
14
330
5
100
AcDbSymbolTableRecord
100
AcDbLinetypeTableRecord
  2
BYBLOCK
 70
     0
  3

 72
    65
 73
     0
 40
0.0
  0
LTYPE
  5
15
330
5
100
AcDbSymbolTableRecord
100
AcDbLinetypeTableRecord
  2
BYLAYER
 70
     0
  3

 72
    65
 73
     0
 40
0.0
  0
LTYPE
  5
16
330
5
100
AcDbSymbolTableRecord
100
AcDbLinetypeTableRecord
  2
CONTINUOUS
 70
     0
  3
Solid line
 72
    65
 73
     0
 40
0.0
  0
ENDTAB
  0
TABLE
  2
LAYER
  5
2
100
AcDbSymbolTable
 70
1
  0
LAYER
  5
50
100
AcDbSymbolTableRecord
100
AcDbLayerTableRecord
  2
0
 70
0
  6
CONTINUOUS
  0
ENDTAB
  0
TABLE
  2
STYLE
  5
3
330
0
100
AcDbSymbolTable
 70
     1
  0
STYLE
  5
11
330
3
100
AcDbSymbolTableRecord
100
AcDbTextStyleTableRecord
  2
STANDARD
 70
     0
 40
0.0
 41
1.0
 50
0.0
 71
     0
 42
2.5
  3
txt
  4

  0
ENDTAB
  0
TABLE
  2
VIEW
  5
6
330
0
100
AcDbSymbolTable
 70
     0
  0
ENDTAB
  0
TABLE
  2
UCS
  5
7
330
0
100
AcDbSymbolTable
 70
     0
  0
ENDTAB
  0
TABLE
  2
APPID
  5
9
330
0
100
AcDbSymbolTable
 70
     2
  0
APPID
  5
12
330
9
100
AcDbSymbolTableRecord
100
AcDbRegAppTableRecord
  2
ACAD
 70
     0
  0
ENDTAB
  0
TABLE
  2
DIMSTYLE
  5
A
330
0
100
AcDbSymbolTable
 70
     1
  0
DIMSTYLE
105
27
330
A
100
AcDbSymbolTableRecord
100
AcDbDimStyleTableRecord
  2
ISO-25
 70
     0
  3

  4

  5

  6

  7

 40
1.0
 41
2.5
 42
0.625
 43
3.75
 44
1.25
 45
0.0
 46
0.0
 47
0.0
 48
0.0
140
2.5
141
2.5
142
0.0
143
0.03937007874016
144
1.0
145
0.0
146
1.0
147
0.625
 71
     0
 72
     0
 73
     0
 74
     0
 75
     0
 76
     0
 77
     1
 78
     8
170
     0
171
     3
172
     1
173
     0
174
     0
175
     0
176
     0
177
     0
178
     0
270
     2
271
     2
272
     2
273
     2
274
     3
340
11
275
     0
280
     0
281
     0
282
     0
283
     0
284
     8
285
     0
286
     0
287
     3
288
     0
  0
ENDTAB
  0
TABLE
  2
BLOCK_RECORD
  5
1
330
0
100
AcDbSymbolTable
 70
     1
  0
BLOCK_RECORD
  5
1F
330
1
100
AcDbSymbolTableRecord
100
AcDbBlockTableRecord
  2
*MODEL_SPACE
  0
BLOCK_RECORD
  5
1B
330
1
100
AcDbSymbolTableRecord
100
AcDbBlockTableRecord
  2
*PAPER_SPACE
  0
ENDTAB
  0
ENDSEC
  0
SECTION
  2
BLOCKS
  0
BLOCK
  5
20
330
1F
100
AcDbEntity
  8
0
100
AcDbBlockBegin
  2
*MODEL_SPACE
 70
     0
 10
0.0
 20
0.0
 30
0.0
  3
*MODEL_SPACE
  1

  0
ENDBLK
  5
21
330
1F
100
AcDbEntity
  8
0
100
AcDbBlockEnd
  0
BLOCK
  5
1C
330
1B
100
AcDbEntity
 67
     1
  8
0
100
AcDbBlockBegin
  2
*PAPER_SPACE
  1

  0
ENDBLK
  5
1D
330
1B
100
AcDbEntity
 67
     1
  8
0
100
AcDbBlockEnd
  0
ENDSEC
  0
SECTION
  2
ENTITIES
  0
LWPOLYLINE
  5
100
100
AcDbEntity
  8
0
 62
7
100
AcDbPolyline
 90
4
 70
1
 10
0.000000
 20
75.000000
 30
0.0
 10
75.000000
 20
75.000000
 30
0.0
 10
75.000000
 20
0.000000
 30
0.0
 10
0.000000
 20
0.000000
 30
0.0
  0
SPLINE
  5
101
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
153.750000
 20
75.000000
 30
0.0
 10
156.733686
 20
75.000000
 30
0.0
 10
159.595167
 20
73.814736
 30
0.0
 10
161.704951
 20
71.704951
 30
0.0
  0
SPLINE
  5
102
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
161.704951
 20
71.704951
 30
0.0
 10
163.814736
 20
69.595167
 30
0.0
 10
165.000000
 20
66.733686
 30
0.0
 10
165.000000
 20
63.750000
 30
0.0
  0
LWPOLYLINE
  5
103
100
AcDbEntity
  8
0
 62
7
100
AcDbPolyline
 90
2
 70
0
 10
101.250000
 20
75.000000
 30
0.0
 10
153.750000
 20
75.000000
 30
0.0
  0
SPLINE
  5
104
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
165.000000
 20
11.250000
 30
0.0
 10
165.000000
 20
8.266314
 30
0.0
 10
163.814736
 20
5.404833
 30
0.0
 10
161.704951
 20
3.295049
 30
0.0
  0
SPLINE
  5
105
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
161.704951
 20
3.295049
 30
0.0
 10
159.595167
 20
1.185264
 30
0.0
 10
156.733686
 20
0.000000
 30
0.0
 10
153.750000
 20
0.000000
 30
0.0
  0
LWPOLYLINE
  5
106
100
AcDbEntity
  8
0
 62
7
100
AcDbPolyline
 90
2
 70
0
 10
165.000000
 20
63.750000
 30
0.0
 10
165.000000
 20
11.250000
 30
0.0
  0
SPLINE
  5
107
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
101.250000
 20
0.000000
 30
0.0
 10
98.266314
 20
0.000000
 30
0.0
 10
95.404833
 20
1.185264
 30
0.0
 10
93.295049
 20
3.295049
 30
0.0
  0
SPLINE
  5
108
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
93.295049
 20
3.295049
 30
0.0
 10
91.185264
 20
5.404833
 30
0.0
 10
90.000000
 20
8.266314
 30
0.0
 10
90.000000
 20
11.250000
 30
0.0
  0
LWPOLYLINE
  5
109
100
AcDbEntity
  8
0
 62
7
100
AcDbPolyline
 90
2
 70
0
 10
153.750000
 20
0.000000
 30
0.0
 10
101.250000
 20
0.000000
 30
0.0
  0
SPLINE
  5
10a
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
90.000000
 20
63.750000
 30
0.0
 10
90.000000
 20
66.733686
 30
0.0
 10
91.185264
 20
69.595167
 30
0.0
 10
93.295049
 20
71.704951
 30
0.0
  0
SPLINE
  5
10b
100
AcDbEntity
  8
0
 62
7
100
AcDbSpline
 70
8
 71
3
 72
8
 73
4
 74
0
 40
0
 40
0
 40
0
 40
0
 40
1
 40
1
 40
1
 40
1
 10
93.295049
 20
71.704951
 30
0.0
 10
95.404833
 20
73.814736
 30
0.0
 10
98.266314
 20
75.000000
 30
0.0
 10
101.250000
 20
75.000000
 30
0.0
  0
LWPOLYLINE
  5
10c
100
AcDbEntity
  8
0
 62
7
100
AcDbPolyline
 90
2
 70
0
 10
90.000000
 20
11.250000
 30
0.0
 10
90.000000
 20
63.750000
 30
0.0
  0
LWPOLYLINE
  5
10d
100
AcDbEntity
  8
0
 62
7
100
AcDbPolyline
 90
1
 70
1
 10
101.250000
 20
75.000000
 30
0.0
  0
ENDSEC
  0
SECTION
  2
OBJECTS
  0
DICTIONARY
  5
C
330
0
100
AcDbDictionary
  3
ACAD_GROUP
350
D
  3
ACAD_MLINESTYLE
350
17
  0
DICTIONARY
  5
D
330
C
100
AcDbDictionary
  0
DICTIONARY
  5
1A
330
C
100
AcDbDictionary
  0
DICTIONARY
  5
17
330
C
100
AcDbDictionary
  3
STANDARD
350
18
  0
DICTIONARY
  5
19
330
C
100
AcDbDictionary
  0
ENDSEC
  0
EOF
