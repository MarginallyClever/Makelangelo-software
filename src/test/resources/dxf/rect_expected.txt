[TOOL R0 G0 B0 A255 D1.000, TRAVEL X0.000 Y75.000, DRAW_LINE X75.000 Y75.000, DRAW_LINE X75.000 Y0.000, DRAW_LINE X0.000 Y0.000, DRAW_LINE X0.000 Y75.000, TRAVEL X101.250 Y75.000, DRAW_LINE X153.750 Y75.000, TRAVEL X165.000 Y63.750, DRAW_LINE X165.000 Y11.250, TRAVEL X153.750 Y0.000, DRAW_LINE X101.250 Y0.000, TRAVEL X90.000 Y11.250, DRAW_LINE X90.000 Y63.750, TRAVEL X101.250 Y75.000, DRAW_LINE X101.250 Y75.000, TRAVEL X153.750 Y75.000, DRAW_LINE X154.048 Y74.996, DRAW_LINE X154.345 Y74.984, DRAW_LINE X154.641 Y74.965, DRAW_LINE X154.935 Y74.937, DRAW_LINE X155.229 Y74.902, DRAW_LINE X155.521 Y74.860, DRAW_LINE X155.811 Y74.810, DRAW_LINE X156.099 Y74.752, DRAW_LINE X156.385 Y74.687, DRAW_LINE X156.670 Y74.615, DRAW_LINE X156.952 Y74.535, DRAW_LINE X157.231 Y74.448, DRAW_LINE X157.509 Y74.354, DRAW_LINE X157.783 Y74.252, DRAW_LINE X158.055 Y74.144, DRAW_LINE X158.324 Y74.028, DRAW_LINE X158.590 Y73.906, DRAW_LINE X158.853 Y73.776, DRAW_LINE X159.112 Y73.640, DRAW_LINE X159.368 Y73.497, DRAW_LINE X159.620 Y73.347, DRAW_LINE X159.869 Y73.191, DRAW_LINE X160.113 Y73.027, DRAW_LINE X160.354 Y72.858, DRAW_LINE X160.590 Y72.682, DRAW_LINE X160.822 Y72.499, DRAW_LINE X161.050 Y72.310, DRAW_LINE X161.273 Y72.115, DRAW_LINE X161.491 Y71.913, DRAW_LINE X161.705 Y71.705, DRAW_LINE X161.913 Y71.491, DRAW_LINE X162.115 Y71.273, DRAW_LINE X162.310 Y71.050, DRAW_LINE X162.499 Y70.822, DRAW_LINE X162.682 Y70.590, DRAW_LINE X162.858 Y70.354, DRAW_LINE X163.027 Y70.113, DRAW_LINE X163.191 Y69.869, DRAW_LINE X163.347 Y69.620, DRAW_LINE X163.497 Y69.368, DRAW_LINE X163.640 Y69.112, DRAW_LINE X163.776 Y68.853, DRAW_LINE X163.906 Y68.590, DRAW_LINE X164.028 Y68.324, DRAW_LINE X164.144 Y68.055, DRAW_LINE X164.252 Y67.783, DRAW_LINE X164.354 Y67.509, DRAW_LINE X164.448 Y67.231, DRAW_LINE X164.535 Y66.952, DRAW_LINE X164.615 Y66.670, DRAW_LINE X164.687 Y66.385, DRAW_LINE X164.752 Y66.099, DRAW_LINE X164.810 Y65.811, DRAW_LINE X164.860 Y65.521, DRAW_LINE X164.902 Y65.229, DRAW_LINE X164.937 Y64.935, DRAW_LINE X164.965 Y64.641, DRAW_LINE X164.984 Y64.345, DRAW_LINE X164.996 Y64.048, DRAW_LINE X165.000 Y63.750, TRAVEL X165.000 Y11.250, DRAW_LINE X164.996 Y10.952, DRAW_LINE X164.984 Y10.655, DRAW_LINE X164.965 Y10.359, DRAW_LINE X164.937 Y10.065, DRAW_LINE X164.902 Y9.771, DRAW_LINE X164.860 Y9.479, DRAW_LINE X164.810 Y9.189, DRAW_LINE X164.752 Y8.901, DRAW_LINE X164.687 Y8.615, DRAW_LINE X164.615 Y8.330, DRAW_LINE X164.535 Y8.048, DRAW_LINE X164.448 Y7.769, DRAW_LINE X164.354 Y7.491, DRAW_LINE X164.252 Y7.217, DRAW_LINE X164.144 Y6.945, DRAW_LINE X164.028 Y6.676, DRAW_LINE X163.906 Y6.410, DRAW_LINE X163.776 Y6.147, DRAW_LINE X163.640 Y5.888, DRAW_LINE X163.497 Y5.632, DRAW_LINE X163.347 Y5.380, DRAW_LINE X163.191 Y5.131, DRAW_LINE X163.027 Y4.887, DRAW_LINE X162.858 Y4.646, DRAW_LINE X162.682 Y4.410, DRAW_LINE X162.499 Y4.178, DRAW_LINE X162.310 Y3.950, DRAW_LINE X162.115 Y3.727, DRAW_LINE X161.913 Y3.509, DRAW_LINE X161.705 Y3.295, DRAW_LINE X161.491 Y3.087, DRAW_LINE X161.273 Y2.885, DRAW_LINE X161.050 Y2.690, DRAW_LINE X160.822 Y2.501, DRAW_LINE X160.590 Y2.318, DRAW_LINE X160.354 Y2.142, DRAW_LINE X160.113 Y1.973, DRAW_LINE X159.869 Y1.809, DRAW_LINE X159.620 Y1.653, DRAW_LINE X159.368 Y1.503, DRAW_LINE X159.112 Y1.360, DRAW_LINE X158.853 Y1.224, DRAW_LINE X158.590 Y1.094, DRAW_LINE X158.324 Y0.972, DRAW_LINE X158.055 Y0.856, DRAW_LINE X157.783 Y0.748, DRAW_LINE X157.509 Y0.646, DRAW_LINE X157.231 Y0.552, DRAW_LINE X156.952 Y0.465, DRAW_LINE X156.670 Y0.385, DRAW_LINE X156.385 Y0.313, DRAW_LINE X156.099 Y0.248, DRAW_LINE X155.811 Y0.190, DRAW_LINE X155.521 Y0.140, DRAW_LINE X155.229 Y0.098, DRAW_LINE X154.935 Y0.063, DRAW_LINE X154.641 Y0.035, DRAW_LINE X154.345 Y0.016, DRAW_LINE X154.048 Y0.004, DRAW_LINE X153.750 Y0.000, TRAVEL X101.250 Y0.000, DRAW_LINE X100.952 Y0.004, DRAW_LINE X100.655 Y0.016, DRAW_LINE X100.359 Y0.035, DRAW_LINE X100.065 Y0.063, DRAW_LINE X99.771 Y0.098, DRAW_LINE X99.479 Y0.140, DRAW_LINE X99.189 Y0.190, DRAW_LINE X98.901 Y0.248, DRAW_LINE X98.615 Y0.313, DRAW_LINE X98.330 Y0.385, DRAW_LINE X98.048 Y0.465, DRAW_LINE X97.769 Y0.552, DRAW_LINE X97.491 Y0.646, DRAW_LINE X97.217 Y0.748, DRAW_LINE X96.945 Y0.856, DRAW_LINE X96.676 Y0.972, DRAW_LINE X96.410 Y1.094, DRAW_LINE X96.147 Y1.224, DRAW_LINE X95.888 Y1.360, DRAW_LINE X95.632 Y1.503, DRAW_LINE X95.380 Y1.653, DRAW_LINE X95.131 Y1.809, DRAW_LINE X94.887 Y1.973, DRAW_LINE X94.646 Y2.142, DRAW_LINE X94.410 Y2.318, DRAW_LINE X94.178 Y2.501, DRAW_LINE X93.950 Y2.690, DRAW_LINE X93.727 Y2.885, DRAW_LINE X93.509 Y3.087, DRAW_LINE X93.295 Y3.295, DRAW_LINE X93.087 Y3.509, DRAW_LINE X92.885 Y3.727, DRAW_LINE X92.690 Y3.950, DRAW_LINE X92.501 Y4.178, DRAW_LINE X92.318 Y4.410, DRAW_LINE X92.142 Y4.646, DRAW_LINE X91.973 Y4.887, DRAW_LINE X91.809 Y5.131, DRAW_LINE X91.653 Y5.380, DRAW_LINE X91.503 Y5.632, DRAW_LINE X91.360 Y5.888, DRAW_LINE X91.224 Y6.147, DRAW_LINE X91.094 Y6.410, DRAW_LINE X90.972 Y6.676, DRAW_LINE X90.856 Y6.945, DRAW_LINE X90.748 Y7.217, DRAW_LINE X90.646 Y7.491, DRAW_LINE X90.552 Y7.769, DRAW_LINE X90.465 Y8.048, DRAW_LINE X90.385 Y8.330, DRAW_LINE X90.313 Y8.615, DRAW_LINE X90.248 Y8.901, DRAW_LINE X90.190 Y9.189, DRAW_LINE X90.140 Y9.479, DRAW_LINE X90.098 Y9.771, DRAW_LINE X90.063 Y10.065, DRAW_LINE X90.035 Y10.359, DRAW_LINE X90.016 Y10.655, DRAW_LINE X90.004 Y10.952, DRAW_LINE X90.000 Y11.250, TRAVEL X90.000 Y63.750, DRAW_LINE X90.004 Y64.048, DRAW_LINE X90.016 Y64.345, DRAW_LINE X90.035 Y64.641, DRAW_LINE X90.063 Y64.935, DRAW_LINE X90.098 Y65.229, DRAW_LINE X90.140 Y65.521, DRAW_LINE X90.190 Y65.811, DRAW_LINE X90.248 Y66.099, DRAW_LINE X90.313 Y66.385, DRAW_LINE X90.385 Y66.670, DRAW_LINE X90.465 Y66.952, DRAW_LINE X90.552 Y67.231, DRAW_LINE X90.646 Y67.509, DRAW_LINE X90.748 Y67.783, DRAW_LINE X90.856 Y68.055, DRAW_LINE X90.972 Y68.324, DRAW_LINE X91.094 Y68.590, DRAW_LINE X91.224 Y68.853, DRAW_LINE X91.360 Y69.112, DRAW_LINE X91.503 Y69.368, DRAW_LINE X91.653 Y69.620, DRAW_LINE X91.809 Y69.869, DRAW_LINE X91.973 Y70.113, DRAW_LINE X92.142 Y70.354, DRAW_LINE X92.318 Y70.590, DRAW_LINE X92.501 Y70.822, DRAW_LINE X92.690 Y71.050, DRAW_LINE X92.885 Y71.273, DRAW_LINE X93.087 Y71.491, DRAW_LINE X93.295 Y71.705, DRAW_LINE X93.509 Y71.913, DRAW_LINE X93.727 Y72.115, DRAW_LINE X93.950 Y72.310, DRAW_LINE X94.178 Y72.499, DRAW_LINE X94.410 Y72.682, DRAW_LINE X94.646 Y72.858, DRAW_LINE X94.887 Y73.027, DRAW_LINE X95.131 Y73.191, DRAW_LINE X95.380 Y73.347, DRAW_LINE X95.632 Y73.497, DRAW_LINE X95.888 Y73.640, DRAW_LINE X96.147 Y73.776, DRAW_LINE X96.410 Y73.906, DRAW_LINE X96.676 Y74.028, DRAW_LINE X96.945 Y74.144, DRAW_LINE X97.217 Y74.252, DRAW_LINE X97.491 Y74.354, DRAW_LINE X97.769 Y74.448, DRAW_LINE X98.048 Y74.535, DRAW_LINE X98.330 Y74.615, DRAW_LINE X98.615 Y74.687, DRAW_LINE X98.901 Y74.752, DRAW_LINE X99.189 Y74.810, DRAW_LINE X99.479 Y74.860, DRAW_LINE X99.771 Y74.902, DRAW_LINE X100.065 Y74.937, DRAW_LINE X100.359 Y74.965, DRAW_LINE X100.655 Y74.984, DRAW_LINE X100.952 Y74.996, DRAW_LINE X101.250 Y75.000]